nonprofit-board-management.php

```diff
@@ -750,18 +750,12 @@ public function display_extensions_sidebar(){
      * Remove the help tab in the top right for all board members.
      *
      * We remove the help tab because we want them to use the support menu we created.
-     *
-     * @param string $contextual_help Help to display in tab that we leave blank.
-     * @param string $screen_id ID of the current admin screen.
-     * @param object $screen Information on the current screen you're viewing.
-     * @return $contextual_help We return an empty string since we don't use help tabs.
      */
-    public function remove_help_tabs( $contextual_help, $screen_id, $screen ){
+    public function remove_help_tabs(){
       if( current_user_can( 'board_member' ) ){
+		$screen = get_current_screen();
```

Comment:
For me the indentation here doesn't line up with the line below in VS Code. If it looks good for you then I'd say leave it alone.